notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.        data_source_csv_filenames=data_source_csv_filenames,
</pre><p>Could we also set data_source_bigquery_table_path=data_source_bigquery_table_path,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Done.